src/server/Group.cpp

```diff
@@ -50,22 +46,21 @@ void Group::refresh() {
 
 void Group::addMember(std::shared_ptr<Player> player) {
     mMembers.push_back(player);
-    mSize = mMembers.size();
-    mCircle->setRadius(mSize * GROUP_SIZE);
-}
-
-int Group::getId() const {
-    return mId;
+    mCircle->setRadius(mMembers.size() * GROUP_SIZE);
```

Comment:
what is GROUP_SIZE supposed to be?  a multiplier for the radius?

Reply:
addressed in 3d212dd